core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendConf.scala

```diff
@@ -262,9 +264,12 @@ object SharedBackendConf {
   /** Login configuration file. */
   val PROP_LOGIN_CONF = ("spark.ext.h2o.login.conf", None)
 
-  /** Override user name for cluster. */
+  /** User name for cluster and the client authentication. */
   val PROP_USER_NAME = ("spark.ext.h2o.user.name", None)
 
+  /** Password for the client authentication. */
+  val PROP_PASSWORD = ("spark.ext.h2o.user.name", None)
```

Comment:
Shouldn't the property be different from `"spark.ext.h2o.user.name"`?

Reply:
As we have requirement that the field `username` needs to be the same as the username passed to the backend, I think it would make sense to reuse this option to limit number of options